pkg/migration/runner_test.go

```diff
@@ -1224,6 +1225,7 @@ func TestE2EBinlogSubscribingCompositeKey(t *testing.T) {
 	assert.NotNil(t, chunk)
 	assert.Equal(t, "((`id1` < 1001)\n OR (`id1` = 1001 AND `id2` < 1))", chunk.String())
 	assert.NoError(t, m.copier.CopyChunk(context.TODO(), chunk))
+	assert.Equal(t, Progress{CurrentState: "copyRows", Summary: "1000/1200 83.33% copyRows ETA TBD"}, m.GetProgress())
```

Comment:
In my experience, tests on raw text output are extremely fragile and become problematic. One character changes, the tests fail, but a future engineer won't know if it's just a text diff or if there's a real issue. If there's a different way to check this, it'd be worth the time/effort to do.

Reply:
There are constants for the `CurrentState`. I can always change that. There are no constants for the summary though.